corehq/apps/cloudcare/templates/cloudcare/formplayer_home.html

```diff
@@ -102,27 +102,31 @@
       <div class="d-lg-none">
         <div id="mobile-restore-as-region"></div>
       </div>
-      <div id="breadcrumb-region" class="print-container sticky-top"></div>
+      <div id="breadcrumb-region" class="print-container"></div>
       <section id="cloudcare-notifications" class="container notifications-container"></section>
       <div class="container case-tile-container">
         <div id="persistent-case-tile" class="print-container"></div>
       </div>
-      <div class="d-lg-flex">
-        <div id="sidebar-region" class="noprint-sub-container mt-3"></div>
-        <div id="content-container" class="container mt-3">
```

Comment:
this `container` is necessary to prevent wide elements from overflowing the div.  That was the issue addressed by https://github.com/dimagi/commcare-hq/pull/35069/commits/177ccc7b119a61df73d46fce3fb314cbd40999a7

I see you put a max-width on the element, which should probably address that, though does that also work well on smaller screen sizes?